lib/crypt.php

```diff
@@ -76,18 +76,18 @@ public static function plaintext($string, $key) {
         $string = base64_decode($string);
 
         /* bail if the crypt text is invalid */
-        if (!$string || strlen($string) <= 200) {
+        if (!$string || mb_strlen($string, '8bit') <= 200) {
```

Comment:
Why 8bit? If the encoded string is something only cypht does, we can keep the old non-mb functions as they are 8bit by default, aren't they?

Reply:
I assume this would be true for all 8bit usages in the following code - if these are hashes that are not multibyte strings, simply use the non-mb functions.

Reply:
Sure, they are. I've replaced the 8bit usages with non-mb functions.

Thanks!